### DISCUSSION TITLE
geyser_mappings is blank , pls help

### DISCUSSION THREAD

**Discussion by luongcongtandpt** [Kas-tle/java2bedrock.sh#10894] [Converter Help]
hi , this is my issue link : https://github.com/Hahaa13/java2bedrock.sh/actions/runs/5840565585
geyser_mapping file is blank , how can i fix this
did i do sthing wrong , my pack is error or the converter error itself ?
pls help me , thank you 
![image](https://github.com/Kas-tle/java2bedrock.sh/assets/137200888/c99c8803-3b38-4f9a-a1e7-06b71536ac92)

**Comment by Rakanlahganteng** [comment]
Same as me

**Comment by LamzyMC** [comment]
So what is the solution?
